### ISSUE TITLE
Toggle high visibility mode only works after pressing some key after

### ISSUE THREAD

**Issue by rogeriodec** [facelessuser/BracketHighlighter#589]
## Description

`ctrl+alt+super+v`  will show the underlines only after pressing some key after
I'm using ST4.

## Steps to Reproduce Issue

1. Create any content in brackets, example, c++ `{ asdlkasdlkasdj }`
2. Goto inside and press `ctrl+alt+super+v` (nothing happens)
3. press any other key and the underlines will appear

**Comment by facelessuser** [comment]
Yes, this enables the mode, but does not immediately force the highlighting to be re-run. On moving the cursor, the highlight would then be run.

**Comment by rogeriodec** [comment]
Okay. But I think if I opened this issue, it's because it didn't seem like a standard behavior.
If you accept my suggestion, make the result visually appear immediately after activating the function, and don't wait for another key, as this creates confusion.

**Comment by facelessuser** [comment]
> Okay. But I think if I opened this issue, it's because it didn't seem like a standard behavior.

I'm simply explaining this is the expected behavior. If you'd like some additional behavior, it is fine to suggest it as long as it is done in a decent manner.

> If you accept my suggestion, make the result visually appear immediately after activating the function, and don't wait for another key, as this creates confusion.

BH is driven by events. That includes edits and cursor movements. BH can also be executed directly. I'm sure you could also chain the "toggle" command with the BH execution command. In Sublime build 4103, it claims to have added a way to chain multiple commands, probably very similar to the existing Chain of Command plugin https://packagecontrol.io/packages/Chain%20of%20Command.

I would assume the suggestion above is sufficient to get you exactly what you would like using built-in Sublime command definitions. If you find it is not, it is possible we could add such functionality. Currently, the command acts exactly as it was designed, it is simply a command to toggle high visibility mode.

Anyways, you are free to suggest us moving forward with the proposed modification if you find the above direction is not acceptable; in which case, I can evaluate the possibility of prioritizing and potentially adding it at some future time.

**Comment by facelessuser** [comment]
As there has been no response for months as to whether our suggestion helped or not or is sufficient, closing the issue as stale.
